### PR TITLE
porting over work from personal repo

### PR DESCRIPTION
**Pull request by risingBirdSong**

### CODE REVIEW

**Comment by risingBirdSong** [comment]
@paullferguson Hey Paul, no rush on this, it's a bigger PR because i'm porting over work i had done on my personal repo, which thankfully Ha noticed. 

It's basically a basic server and simple react app to display data of one item at a time.

**Comment by paullferguson** [comment]
You can remove all the `.gitkeep` files now that the directories have files in them

File: client/components/HelloWorld.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (paullferguson) [inline client/components/HelloWorld.jsx:1]: I think you can get rid of this file now ;)

File: client/components/itemdetail.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  //   });
+  // }
+
+  render() {
+    const { product } = this.state;
+    let display;
+    if (product) {
+      display = (
+        <div>
+          <h2>{product.producer}</h2>
```
Comment (paullferguson) [inline client/components/itemdetail.jsx:38]: 🙌

File: database/index.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    type: Number,
+    default: 0,
+  },
+  wasPrice: Number,
+  currentPrice: {
+    type: Number,
+    required: true,
+    default: 10,
+  },
+  // Paul pointed out this can be calc... on front end, further not, he was trolling...
```
Comment (paullferguson) [inline database/index.js:47]: :trollface: 
but also...

**Comment by risingBirdSong** [comment]
@paullferguson Hi, thanks for the catches... i've reviewed the changes and made those changes... 
goodbye world :(

**Pull request merged**